src/main/webapp/app/admin/standardized-competencies/import/admin-import-standardized-competencies.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="container">
+    <h2 jhiTranslate="artemisApp.standardizedCompetency.manage.import.title"></h2>
+    <div class="card mt-3">
+        <div class="card-header d-flex align-items-center mw-100 clickable" (click)="toggleCollapse()">
+            <div class="icon-container me-2">
+                <fa-icon [icon]="faChevronRight" class="rotate-icon chevron-position" [class.rotated]="!isCollapsed" />
+            </div>
+            <h5 class="mb-0" jhiTranslate="artemisApp.standardizedCompetency.manage.import.format.title"></h5>
+        </div>
+        <div class="card-body" [ngbCollapse]="isCollapsed">
+            <p jhiTranslate="artemisApp.standardizedCompetency.manage.import.format.text1"></p>
+            <p jhiTranslate="artemisApp.standardizedCompetency.manage.import.format.text2"></p>
+            <p jhiTranslate="artemisApp.standardizedCompetency.manage.import.format.text3"></p>
+            <div [innerHTML]="importExample | htmlForMarkdown"></div>
+        </div>
+    </div>
+    <input class="mt-3" type="file" accept="application/json" (change)="onFileChange($event)" />
+    <h4 class="mt-3" jhiTranslate="artemisApp.standardizedCompetency.manage.import.preview.title"></h4>
+    @if (count && count?.knowledgeAreas) {
```

Comment:
Maybe you could rename this field to `importCount` - just something I stumbled over when reviewing